nominal/sdk.py

```diff
@@ -271,6 +279,108 @@ def _from_conjure(cls, client: NominalClient, dataset: scout_catalog.EnrichedDat
             _client=client,
         )
 
+@dataclass(frozen=True)
+class LogSet:
```

Comment:
I called it `LogSet` because I added methods inside to do stuff with it. I figure that's better than keeping with the conjure definition of `LogSetMetadata`, and then adding attach logs / search logs to the client itself